public/class-award-force-sso.php

```diff
@@ -59,7 +60,11 @@ private function requestSlug(WP_User $user)
     {
         $response = $this->requestSlugByEmail($user->user_email);
 
-        if ($response->slug) {
+        if (isset($response->status_code) && $response->status_code != 200) {
```

Comment:
@nicocucuzza as per the thread in slack... I think these checks here will mean it will never get passed this point and create the user if it doesn't exist.